.ci/Makefile

```diff
@@ -0,0 +1,102 @@
+SHELL = /bin/bash -eo pipefail
+
+AWS_FOLDER = aws
+
+export AWS_FOLDER
+
+build:
+	rm -rf ../$(AWS_FOLDER) || true
+	mkdir -p ../$(AWS_FOLDER)
+	cd ../$(AWS_FOLDER); \
+		npm init -y; \
+		npm install --global-style https://github.com/elastic/apm-agent-nodejs#$(BRANCH_NAME); \
+		mkdir nodejs; \
+		mv node_modules nodejs
+env:
+	env
+dist: validate-branch-name build
+	rm -f $(BRANCH_NAME).zip || true
+	cd ../$(AWS_FOLDER) ; \
+		zip -r $(BRANCH_NAME).zip nodejs
+
+# List all the AWS regions
+get-all-aws-regions:
+	@aws \
+		ec2 \
+		describe-regions \
+		--region us-east-1 \
+		--output json \
+		--no-cli-pager \
+		| jq -r '.Regions[].RegionName' > .regions
+
+# Publish the given LAYER in all the AWS regions
+publish-in-all-aws-regions: validate-layer-name get-all-aws-regions
+	@mkdir -p $(AWS_FOLDER)
+	@while read AWS_DEFAULT_REGION; do \
+		echo "publish '$(ELASTIC_LAYER_NAME)' in $${AWS_DEFAULT_REGION}"; \
+		AWS_DEFAULT_REGION="$${AWS_DEFAULT_REGION}" ELASTIC_LAYER_NAME=$(ELASTIC_LAYER_NAME) $(MAKE) publish > $(AWS_FOLDER)/$${AWS_DEFAULT_REGION}; \
+		AWS_DEFAULT_REGION="$${AWS_DEFAULT_REGION}" ELASTIC_LAYER_NAME=$(ELASTIC_LAYER_NAME) $(MAKE) grant-public-layer-access; \
+	done <.regions
+
+# Publish the given LAYER in the given AWS region
+publish: validate-layer-name validate-aws-default-region
+	@aws lambda \
+		--output json \
+		publish-layer-version \
+		--layer-name "$(ELASTIC_LAYER_NAME)" \
+		--description "AWS Lambda Extension Layer for the Elastic APM Node.js Agent" \
+		--license "Apache-2.0" \
+		--compatible-runtimes nodejs14.x nodejs12.x nodejs10.x \
+		--zip-file "fileb://../$(AWS_FOLDER)/$(BRANCH_NAME).zip"
+
+# Grant public access to the given LAYER in the given AWS region
+grant-public-layer-access: validate-layer-name validate-aws-default-region
+	@echo "[debug] $(ELASTIC_LAYER_NAME) with version: $$($(MAKE) -s --no-print-directory get-version)"
+	@aws lambda \
+		--output json \
+		add-layer-version-permission \
+		--layer-name "$(ELASTIC_LAYER_NAME)" \
+		--action lambda:GetLayerVersion \
+		--principal '*' \
+		--statement-id "$(ELASTIC_LAYER_NAME)" \
+		--version-number $$($(MAKE) -s --no-print-directory get-version) > $(AWS_FOLDER)/.$(AWS_DEFAULT_REGION)-public
+
+# Get the ARN Version for the AWS_REGIONS
+# NOTE: jq -r .Version "$(AWS_FOLDER)/$(AWS_DEFAULT_REGION)" fails in the CI
+#       with 'parse error: Invalid numeric literal at line 1, column 5'
+get-version: validate-aws-default-region
+	@grep '"Version"' "$(AWS_FOLDER)/$(AWS_DEFAULT_REGION)" | cut -d":" -f2 | sed 's/ //g' | cut -d"," -f1
+
+# Generate the file with the ARN entries
+create-arn-file: validate-suffix-arn-file
+	@./scripts/create-arn-table.sh
+
+release-notes: validate-branch-name validate-suffix-arn-file
+	@gh release list
+	@gh \
+		release \
+		create $(BRANCH_NAME) \
+		--title '$(BRANCH_NAME)' \
+		--generate-notes \
```

Comment:
Probably we don't want this, otherwise

<img width="1219" alt="image" src="https://user-images.githubusercontent.com/2871786/156021964-9b8ba12b-fab3-4333-9671-102a27da14f5.png">


```suggestion
```

Reply:
Agreed. I think it would be better to point to the published release notes, e.g. `https://www.elastic.co/guide/en/apm/agent/nodejs/current/release-notes-3.x.html#release-notes-3.29.0`, at the top of the release section, if possible.

Reply:
https://github.com/elastic/apm-agent-nodejs/pull/2585/files#diff-bf80150258607c1e440ef8c2f2b828b033f7d11998cfd62e1f0716b9020acfbdR15 is the implementation and the URL is calculated on the fly based on the `TAG` version.

https://github.com/elastic/apm-agent-nodejs/releases/tag/untagged-67f14d7ca86fb8fa5e49

<img width="907" alt="image" src="https://user-images.githubusercontent.com/2871786/156208675-ec3c7ec8-bbb9-465a-9b7e-3e8506e58d63.png">